bootstrap.php

```diff
@@ -1,78 +1,7 @@
 <?php
 
-use NewfoldLabs\WP\Module\Performance\CacheTypes\Browser;
-use NewfoldLabs\WP\Module\Performance\CacheTypes\Cloudflare;
-use NewfoldLabs\WP\Module\Performance\CacheTypes\File;
-use NewfoldLabs\WP\Module\Performance\CacheTypes\Skip404;
-use NewfoldLabs\WP\Module\Performance\Performance;
-use NewfoldLabs\WP\Module\Performance\ResponseHeaderManager;
-use NewfoldLabs\WP\ModuleLoader\Container;
+namespace NewfoldLabs\WP\Module\Performance;
 
-use function NewfoldLabs\WP\Module\Performance\getCacheLevel;
-use function NewfoldLabs\WP\ModuleLoader\register;
-use function NewfoldLabs\WP\Context\getContext;
+require_once __DIR__ . '/includes/PerformanceFeature.php';
```

Comment:
We should use a Composer file autoloader for this.

Reply:
Got it